iron-request.html

```diff
@@ -229,12 +229,21 @@
         }, this);
       }
 
+      var contentType;
+      if (options.headers) {
+        contentType = options.headers['Content-Type'];
+      }
+      body = this._encodeBodyObject(options.body, contentType);
```

Comment:
Is this a global leak? I can't find the declaration.

I have a somewhat bike-shed-y opinion that identifiers should be declared where they would get hoisted to. It makes it easier to check for errors like this, for example. What do you think?